angular/src/app/home/home.component.html

```diff
@@ -0,0 +1,11 @@
+<br>
+<p> Free and diverse media is necessary for a well-informed public and a functioning society.
+However, objective reporting is not possible and sometimes newspapers even favor a certain ideology or political direction. </p>
+<p>
+Even though we are not consciously aware of the presence of reporting biases, such as the tendency to favor arguments of a political party or to constantly set a country in a negative context, they do have significant influence on our world view. </p>
+<p>
+Knowing these biases helps to critically reflect the reliability and trustworthiness of an article.</p><p>
```

Comment:
being consistent in writing down code makes it easier to read. I would stick to the  each `<p>` or `</p>` on a seperate line

Reply:
+1